src/clojure_tictactoe/cli/output/instructions_printer.clj

```diff
@@ -21,7 +21,18 @@ Examples:
   (println welcome-message)
   (println instructions-message))
 
-
-
 (defn print-marker-instructions [player]
   (println (str "What marker would you like for " player "?")))
+
+(defn stringify-options [options]
```

Comment:
This name is a bit misleading since you're doing more than just converting the list to a string.

Reply:
Good point.